packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -179,113 +156,115 @@ function ChatWidget(props: ChatWidgetProps): JSX.Element {
     setInputText(inputText + emoji.emoji)
   }
 
-  const resetInputSize = (): void => {
-    if (!content.current || !inputRow.current) {
-      throw new Error('Cannot find elements for resize')
-    }
-    content.current.style.height = '290px'
-    inputRow.current.style.height = '60px'
-  }
-
   const sendMessage = async (): Promise<void> => {
     await ChatApi.sendChatMessage(teamId, patient.userid, inputText, privateMessage)
     const messages = await ChatApi.getChatMessages(teamId, patient.userid)
     setMessages(messages)
     setInputText('')
-    resetInputSize()
   }
 
-  const inputHandler = (event: React.ChangeEvent<HTMLTextAreaElement>): void => {
-    setInputText(event.target.value)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏So all of this is no longer necessary? How come?